trade-finance-manager-api/src/v1/controllers/auth/auth.controller.js

```diff
@@ -0,0 +1,24 @@
+const authProvider = require('./auth-provider');
+const authService = require('./auth-service');
+
+exports.getLoginUrl = async (req, res, next) => {
+  try {
+    const loginInfo = await authProvider.getLoginUrl(req, res, next);
```

Comment:
I might be misreading this but it seems `getLoginUrl` doesn't need req/res/next

Reply:
yes, I removed them from implementation, but forgot in the call.

Reply:
Have updated this